desc/compute/_basis_vectors.py

```diff
@@ -115,11 +115,7 @@ def _e_sup_rho_t(params, transforms, profiles, data, **kwargs):
             / data["sqrt(g)"] ** 2
         ).T,
         lambda: (
-            (
-                cross(data["e_theta_r"], data["e_zeta_t"])
```

Comment:
This is always 0 anyway so I just remove it.

Reply:
why is it always zero? And would that be true even with a zeta that is not the cylindrical toroidal angle?

Reply:
Yes because e_zeta_t == e_theta_z (i.e. $\partial_{\theta \zeta} \mathbf{e} = \partial_{\zeta \theta} \mathbf{e}\$) and e_theta = 0 everywhere on the magnetic axis.

Reply:
In the cylindrical case: $e_{\theta} = [R_{\theta}, 0, Z_{\theta}]$ while $e_zeta = [R_{\zeta}, R, Z_{\zeta}]$
$\partial_{\zeta}e_{\theta} = [R_{\theta,\zeta}, 0, Z_{\theta,\zeta}]$ while $\partial_{\theta}e_{\zeta} = [R_{\theta,\zeta}, R_{\theta}, Z_{\theta,\zeta}]$

Reply:
Hm ok I see. Thanks for clarifying for me